webrtc-stats.html

```diff
@@ -464,6 +465,23 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>remoteTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Present if <code>isRemote</code> is <code>true</code>,
+                  <code>remoteTimestamp</code>, of type
+                  <code>DOMHighResTimeStamp</code> [[!HIGHRES-TIME]],
+                  represents the remote timestamp at which these statistics
```

Comment:
I'd suggest adding "This timestamp, if present, is derived from the NTP timestamp in an RTCP SR packet, which reflects the remote side's clock. That clock may not be synchronized with the local clock."

I think "derived from" gives enough wiggle room to allow for other computations if the report arrived in RTCP that didn't contain an SR (for instance in RR-only packets).